packages/storybook/stories/Stateful Client/OverviewStateful.stories.mdx

```diff
@@ -136,7 +136,8 @@ For more information on the `ChatThreadClient`, see [ChatThreadClient](https://a
 
 Stateful clients enable developers to access all the APIs that low-level calling and chat SDKs expose.
 For a complete list of capabilities see [Calling Overview](https://docs.microsoft.com/azure/communication-services/concepts/voice-video-calling/calling-sdk-features) and
-[Chat Overview](https://docs.microsoft.com/azure/communication-services/concepts/chat/sdk-features).
+[Chat Overview](https://docs.microsoft.com/azure/communication-services/concepts/chat/sdk-features). For the reference of the interfaces, see [StatefulCallClient interface](https://docs.microsoft.com/javascript/api/@azure/communication-react/statefulcallclient) and
```

Comment:
nit: I think instead of lumping it together in the same paragraph. It may read nicer if we do it at the end of the chart as:
```
For more information see [StatefulCallClient interface](https://docs.microsoft.com/javascript/api/@azure/communication-react/statefulcallclient) and [StatefulChatClient interface](https://docs.microsoft.com/javascript/api/@azure/communication-react/statefulchatclient).
```